boot/freeldr/freeldr/lib/mm/heap.c

```diff
@@ -557,6 +557,8 @@ ExAllocatePoolWithTag(
     return FrLdrHeapAllocateEx(FrLdrDefaultHeap, NumberOfBytes, Tag);
 }
 
+// Deprecated.
+#if 0
```

Comment:
Must this function(s) exist, for potential callers?

Reply:
You can do is something like this
```#define ExAllocatePool ExAllocatePool_is_deprecated_use_ExAllocatePoolWithTag_instead```

Reply:
We do have a version control system. Even deleted code is not lost forever therefore.
I do recommend whenever feeling uncertain whether 'to just comment, because it might be useful later, or whether to delete' the default mode in your head should always favor the delete.
What is just feels slightly difficult for you to do today, will become even harder for other people to decide in the future, which are not familiar with the changes you do today.
Just make tabula rasa here. Dead code needs maintenance too. Just nonexisting code is always correct code.

Reply:
> `#define ExAllocatePool ExAllocatePool_is_deprecated_use_ExAllocatePoolWithTag_instead`

Good idea, but it does not help from this .c file.

Eventually, I will leave these functions in place.
Nonetheless, I submitted #5854.

> We do have a version control system. 

Well, I was looking for an answer, not a lecture, which I don't even agree with.